src/jit/unwindamd64.cpp

```diff
@@ -75,6 +120,37 @@ void Compiler::unwindBegProlog()
     func->unwindHeader.FrameOffset = 0;
 }
 
+#ifdef UNIX_AMD64_ABI
+template <typename T>
+inline
+static T* allocate_any(jitstd::allocator<void>& alloc, size_t count = 1)
+{
+    return jitstd::allocator<T>(alloc).allocate(count);
+}
+typedef jitstd::vector<CFI_CODE> CFICodeVector;
+
+void Compiler::unwindBegPrologCFI()
+{
+    assert(compGeneratingProlog);
+
+    FuncInfoDsc* func = funCurrentFunc();
+
+    // There is only one prolog for a function/funclet, and it comes first. So now is
+    // a good time to initialize all the unwind data structures.
+
+    unwindGetFuncLocations(func, true, &func->startLoc, &func->endLoc);
+
+    if (fgFirstColdBlock != nullptr)
+    {
+        unwindGetFuncLocations(func, false, &func->coldStartLoc, &func->coldEndLoc);
+    }
+
+    jitstd::allocator<void> allocator(getAllocator());
+
+    func->cfiCodes = new (allocate_any<CFICodeVector>(allocator), jitstd::placement_t()) CFICodeVector(allocator);
```

Comment:
I'm not sure what the minimal bar. Many small functions need very small CFIs like 1 ~2. But complex functions with lots of eh could need many. @BruceForstall Can you suggest what the reasonable size for this vectory?